release-team/role-handbooks/communications/templates/pr-email.md

```diff
@@ -0,0 +1,19 @@
+This email must be sent to the following recipients:
+- pr@cncf.io
+- release-comms@kubernetes.io
+- <release-lead email, found in the contact sheet>
+
+```
+Hi there!
+
+I'm the Comms Lead for Kubernetes <release version> (currently scheduled to release <release date>).
+I'm reaching out to get our release day and pre-release press handled.
+
+- 2-3 release highlights: <share 2-3 release highlights>
+- Availability of our Lead for interviews <get it from release lead>
+- the draft blog can be found at this link: https://github.com/kubernetes/website/pull/xxxx <share the draft PR>
+
+Note: the draft blog will not contain the theme and the release logo, those will be added just ahead of the release to avoid spoiling the news but we can share such material in advance and in a separated file with you, as soon as our Lead creates the logo and shares the theme.
```

Comment:
```suggestion
Note: the draft blog will not contain the theme and the release logo, those will be added just ahead of the release to avoid spoiling the news but we can share such material in advance and in a separate file with you, as soon as our Lead creates the logo and shares the theme.
```